osu.Game/Screens/Select/BeatmapDetails.cs

```diff
@@ -50,7 +50,7 @@ public BeatmapInfo Beatmap
 
             set
             {
-                if (beatmap == value) return;
+                if (beatmap == value || value.OnlineBeatmapSetID < 1) return;

```

Comment:
Would it be a better idea to prevent `BeatmapSets` being selected based on their hash values?